src/Uno.UI/LayoutProvider.Android.cs

```diff
@@ -0,0 +1,133 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Android.App;
+using Android.Graphics;
+using Android.Graphics.Drawables;
+using Android.Util;
+using Android.Views;
+using Android.Widget;
+using Windows.Foundation;
+using Rect = Android.Graphics.Rect;
+
+namespace Uno.UI
+{
+	public class LayoutProvider
```

Comment:
Should this be public? Preferably consumer code gets these values via the UWP contract rather than a platform-specific class.